### ISSUE TITLE
[Accessibility][Screen reader]: Name, Role and state is not properly defined for split buttons under 'ContextualMenu with checkable menu items and toggleable split button'. (Bug 4)

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10830]
Tracking all of the issues brought up with the 08/24 accessibility pass on Contextual Menu  control
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bcef6201c-bd85-44b6-a2d3-3784248461e0%7D&action=default&uid=%7BCEF6201C-BD85-44B6-A2D3-3784248461E0%7D&ListItemId=1259&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

Breakout of issue #6360

**Comment by joschect** [comment]
It's unclear how we should define state and role for a split button in general. There doesn't seem to be any aria guidance on how a split button should be read out. This is a problem since it isn't read correctly and could be confusing. It only impacts split buttons in menus though.

**Comment by joschect** [comment]
The regular split button appears to have a similar problem, it's only read out as a split button because of a described by. We'll need more design input for the proper way to communicate a split button, or we should consider deprecating it and eventually removing it.

**Comment by micahgodbolt** [comment]
this little guy gets "Open, split button, collapsed" upon focus

![image](https://user-images.githubusercontent.com/1434956/71284157-f2eb9700-2316-11ea-8a27-f18ad6f682ca.png)

**Comment by micahgodbolt** [comment]
here's what Kendo angular does

https://www.telerik.com/kendo-angular-ui/components/buttons/splitbutton/accessibility/


> The SplitButton uses the aria-label="[button text] splitbutton", role="listbox", aria-haspopup="true", aria-expanded, and aria-owns attributes. The items in the popup are marked with role="menuitem".

**Comment by micahgodbolt** [comment]
not sure why it has listbox on the button. that seems wrong

**Comment by xugao** [comment]
took a look. i think the bug here is we are giving incorrect role to the menu item (should be "menuitemcheckbox" instead of "menuitem" in case it's checkable).
